projects/js-packages/charts/src/types.ts

```diff
@@ -34,16 +34,19 @@ export type DataPointDate = {
 	label?: string;
 };
 
+export type SeriesDataOptions = {
+	gradient?: { from: string; to: string; fromOpacity?: number; toOpacity?: number };
+	stroke?: string;
+	seriesLineStyle?: LineStyles;
+	legendShapeStyle?: CSSProperties;
+	type?: 'comparison';
+};
+
 export type SeriesData = {
 	group?: string;
```

Comment:
so we'll add the matching of comparison matched by the `group` param right? Asking because I'm hoping remove custom comparing styling from Woo Analytics atm.